notifications/src/main/kotlin/com/amazon/opendistroforelasticsearch/commons/notifications/model/Webhook.kt

```diff
@@ -32,7 +32,7 @@ import java.io.IOException
  */
 data class Webhook(
     val url: String
```

Comment:
From what I see the frontend has `Host`, `Port`, `Path`, a list of `parameters` and a list of `headers` for webhook channels, but the backend only stores url string. 

So when the user wants to edit this channel, I can try parsing the url back to get all params but I'm not sure how to get the original webhook headers? I remember we had this discussion before regarding editing webhook channels, any updates? @zhongnansu

Reply:
Host + Port + Path + parameters = url. we will have to check if headers is getting used before adding.